net/model/CurrentItems.h

```diff
@@ -32,6 +32,7 @@
 @property(strong, nonatomic) UIImage *issueImage;
 @property(strong, nonatomic) NSMutableArray<id<UserImageDelegate>> *userImageDelegates;
 @property(strong, nonatomic) NSMutableArray<id<IssueImageDelegate>> *issueImageDelegates;
+@property(strong, nonatomic) NSString *appTitle;
```

Comment:
for now situation is changed. We have single title, so this property is already removed! :)